src/node.cc

```diff
@@ -982,6 +984,41 @@ void SetupNextTick(const FunctionCallbackInfo<Value>& args) {
       FIXED_ONE_BYTE_STRING(args.GetIsolate(), "_setupNextTick"));
 }
 
+void PromiseRejectCallback(PromiseRejectMessage message) {
+  Isolate* isolate = message.GetPromise()->GetIsolate();
+
+  CHECK(isolate);
```

Comment:
You can remove the CHECK, it can never be NULL now. :-)

It would be better to move `Local<Promise> promise = message.GetPromise();` to the top because calling GetPromise() twice is slightly inefficient, it creates a Local twice.